spec/simplecov_spec.rb

```diff
@@ -270,6 +270,31 @@
       end
     end
 
+    context "with_reports_to_clear_up" do
+      after do
+        clear_mergeable_reports
+      end
+
+      def a_void_method_call_that_wont_warn
+        true
+      end
+
+      it "starts coverage of subprocesses" do
+        SimpleCov.command_name "RSpec"
+        SimpleCov.start
```

Comment:
This affects global state (sadly in the design of SimpleCov) - which I guess would make writing another test for this hard and might lead to funky interactions with other tests down the road. But yes, I know you said testing was tricky I'm trying to come up with something better :thinking: